designs/CentralNuGetManagement/CPVM-Project_Package_Version_Override.md

```diff
@@ -0,0 +1,425 @@
+- A `<PackageVersion>` may have a `CentralManagementGroup` defined which allows a `<PackageReference>` to refer to this version using the `CentralManagementGroup` attribute.
+    - When a `Version` to be imported by a `<PackageReference>` needs to be different than the global group defined in `Directory.Packages.props`, a `CentralManagementGroup` can be specified on both the `<PackageReference>` in a project file and a corresponding `<PackageVersion>` in `Directory.Packages.props` that specifies the needed `Version`.
+    - A `<PackageReference>` may only have one `CentralManagementGroup` defined.
+    - A project can include references to multiple `CentralManagementGroup`.
+    - When `CentralManagementGroup` is not specified, both `<PackageVersion>` and `<PackageReference>` reference a global `CentralManagementGroup` that is automatically created.
+- Transitive dependencies are resolved based on finding a common level of specificity between direct imports.
+    - If a transitive dependency is imported by a package defined within the global group, then the `<PackageVersion>` for the transitive dependency from the global group applies.
+    - If a transitive dependency is imported by a package in multiple `CentralManagementGroup` that both include a `<PackageVersion>` for that transitive dependency, then the `<PackageVersion>` for the transitive dependency from the global group applies.
+    - If a transitive dependency is imported by a `<PackageReference>` with a `CentralManagementGroup` specified, then the `<PackageVersion>` for the transitive dependency from the `CentralManagementGroup` applies. If the transitive dependency does not have a matching `<PackageVersion>` within the same `CentralManagementGroup`, then the global group applies.
+    - If a transitive dependency is initiated by multiple packages within multiple, different `CentralManagementGroup` but **only one** of the `CentralManagementGroup` specifies a `<PackageVersion>` for the transitive dependency, then that version is used.
+    - If a transitive dependency is initiated by multiple packages within multiple, different `CentralManagementGroup` and multiple `CentralManagementGroup` specify **the same** `<PackageVersion>` for the transitive dependency, then the common version is used.
+    - If a transitive dependency is initiated by multiple packages within multiple, different `CentralManagementGroup` and multiple `CentralManagementGroup` specify **different** `<PackageVersion>` for the transitive dependency, then the project will throw a build error due to a [cousin dependency](https://docs.microsoft.com/en-us/nuget/concepts/dependency-resolution#cousin-dependencies) conflict.
```

Comment:
Why error? This seems like regular NuGet resolution. In a non CPVM scenario, this is one of the easiest resolution to do. I am concerned that we may be looking at many such errors making the feature unusable.